sf-docker/docker-compose.yml

```diff
@@ -334,6 +334,7 @@ services:
       - RabbitMq__Password=${RabbitMQ__Password}
       - RabbitMq__Port=${RabbitMQ__Port}
       - Kestrel__EndPoints__Http__Url=${Hosting__Host}:${Hosting__Port}
+    env_file: .env.sfac
```

Comment:
same change should be added to jetson-docker-compose.yml
what about variables like RabbitMQ__Password which are defined in .env file? are variables from .env file merged with variables from .env.sfac?

Reply:
I think yes, I am using it this way on SRV-BA-6D and it is working